### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,17 +21,14 @@ Use TiDB as MySQL. You can replace MySQL with TiDB to power your application wit
 Enjoy TiDB as much as we love Go. We believe Go code is both easy and enjoyable to work with. Go makes us improve TiDB fast and makes it easy to dive into the codebase.
 
 
-- __NewSQL over HBase__  
-Turn HBase into NewSQL database
+- __NewSQL over TiKV
```
Comment (qiuyesuifeng) [inline README.md]: Update to `__NewSQL over TiKV __`.

File: README.md [hunk 2]
```diff
@@ -21,17 +21,14 @@ Use TiDB as MySQL. You can replace MySQL with TiDB to power your application wit
 Enjoy TiDB as much as we love Go. We believe Go code is both easy and enjoyable to work with. Go makes us improve TiDB fast and makes it easy to dive into the codebase.
 
 
-- __NewSQL over HBase__  
-Turn HBase into NewSQL database
+- __NewSQL over TiKV 
+Turn TiKV into NewSQL database
```
Comment (qiuyesuifeng) [inline README.md]: Seems `Status` section can be removed.

**Comment by ngaut** [comment]
PTAL

**Comment by shenli** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**